Orthtree/include/CGAL/Orthtree/Node.h

```diff
@@ -44,6 +44,22 @@ struct Node_access
 
   template <typename Node>
   static void split(Node node) { return node.split(); }
+
+  template <typename Node, int D>
```

Comment:
```suggestion
  template <int D, typename Node>
```

`Node` can be deduced